docs/how-to/deploy.md

```diff
@@ -12,33 +12,40 @@ This guide provides an overview of Amplication's deployment options and links to
 
 ## Understanding Amplication's Deployment Architecture
 
-Services created with Amplication, which automatically generates the code, comprise of at least two main parts:
+Amplication services are composed of at least two main parts, all of which are automatically generated by our platform.
 
-1. The server that runs the API
-2. A database
+1. **API Server**
+2. **Database**
 
-There can be other parts like a Kafka message broker, or a Redis cache, other integration that your service needs.
-These will all be additional containers that will be specified in the `docker-compose` file.
+Depending on your service requirements, Amplication can automatically generate the code for additional components such as:
 
-When it comes to deployment, you have several options to choose from, allowing you to select the best approach for your development workflow and production needs.
+- Message brokers like Kafka
+- Caching solutions like Redis
+- Other integrations specific to your service needs
+
+These additional components are seamlessly integrated as containers in your `docker-compose` file.
+
+All components, including optional ones, are auto-generated by Amplication.
 
 ## Deployment Options
 
+When it comes to deployment, you have several options to choose from, allowing you to select the best approach for your development workflow and production needs.
+
 ### Docker
 
-Amplication generates all necessary configuration and scripts for Docker deployment, making it easy to containerize your application.
+With the Docker Deployment plugins, Amplication generates all necessary configuration and scripts for Docker deployment, making it easy to containerize your application.
```

Comment:
Previous suggestion was better- as we have no plugins for docker- see /deploy/docker-desktop for more info